tests/designs/vhdl_zerovector/Makefile

```diff
@@ -0,0 +1,35 @@
+# Copyright cocotb contributors
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# Simulator configuration
+# ----------------------- 
+# Simulator (modelsim is still 32-bit only)
+SIM ?= modelsim
+ARCH ?= i686
+# language of toplevel module
+TOPLEVEL_LANG = vhdl
+# VHDL language version
+VCOM_ARGS += -2008
+# simulation time precision
+SIM_ARGS += -t ns
```

Comment:
I think these need to be hidden behind a line like `ifneq ($(filter $(SIM), questa modelsim),)` - `SIM_ARGS` because it applies to all simulators which either don't understand the `-t` option at all or do not have the same behavior as ModelSim,  and `VCOM_ARGS` because of good style :)